app/app.go

```diff
@@ -479,8 +479,19 @@ func New(
 		appCodec, keys[metadatatypes.StoreKey], app.GetSubspace(metadatatypes.ModuleName), app.AccountKeeper, app.AuthzKeeper, app.AttributeKeeper,
 	)
 
+	markerReqAttrBypassAddrs := []sdk.AccAddress{
+		authtypes.NewModuleAddress(authtypes.FeeCollectorName),     // Allow collecting fees in restricted coins.
+		authtypes.NewModuleAddress(rewardtypes.ModuleName),         // Allow rewards to hold onto restricted coins.
+		authtypes.NewModuleAddress(quarantine.ModuleName),          // Allow quarantine to hold onto restricted coins.
+		authtypes.NewModuleAddress(govtypes.ModuleName),            // Allow restricted coins in deposits.
+		authtypes.NewModuleAddress(distrtypes.ModuleName),          // Allow bond denom to be a restricted coin.
```

Comment:
Technically this is the _fee_ denom and not the _bond_ denom.  In some cases (our mainnet) these two are the same but this is not always true.

Reply:
Updated.